src/components/templates/Camp/components/Map.tsx

```diff
@@ -109,495 +90,156 @@ export const Map: React.FC<MapProps> = ({
-                          <CampAttendance
-                            attendances={attendances}
-                            venue={venue}
-                            room={room}
-                          />
-                        </div>
-                      </div>
-                    )}
-
-                  <div className={`camp-venue-text`}>
-                    {(!venue.roomVisibility ||
-                      venue.roomVisibility === RoomVisibility.nameCount ||
-                      (venue.roomVisibility === RoomVisibility.count &&
-                        hasAttendance)) && (
-                      <div className="camp-venue-maininfo">
-                        {(!venue.roomVisibility ||
-                          venue.roomVisibility ===
-                            RoomVisibility.nameCount) && (
-                          <div className="camp-venue-title">{room.title}</div>
-                        )}
-                        <CampAttendance
-                          attendances={attendances}
-                          venue={venue}
-                          room={room}
-                        />
-                      </div>
-                    )}
-                    <div className="camp-venue-secondinfo">
-                      <div className="camp-venue-desc">
-                        <p>{room.subtitle}</p>
-                        <p>{room.about}</p>
-                      </div>
-                      <div className="camp-venue-actions">
-                        {isExternalUrl(room.url) ? (
-                          <a
-                            className="btn btn-block btn-small btn-primary"
-                            onClick={() => roomEnter(room)}
-                            href={getRoomUrl(room.url)}
-                            target="_blank"
-                            rel="noopener noreferrer"
-                          >
-                            {venue.joinButtonText ?? "Join the room"}
-                          </a>
-                        ) : (
-                          <a
-                            className="btn btn-block btn-small btn-primary"
-                            onClick={() => roomEnter(room)}
-                            href={getRoomUrl(room.url)}
-                          >
-                            {venue.joinButtonText ?? "Join the room"}
-                          </a>
-                        )}
-                      </div>
-                    </div>
-                  </div>
-                </div>
-              </div>
+              <MapRow
+                key={`row${rowIndex}`}
+                showGrid={venue.showGrid}
+                hasSeatedPartygoer={hasSeatedPartygoer}
+                seatedPartygoerIsMe={isMe}
+                // TODO: make this a submodule + useCallback()?
+                onSeatClick={() => onSeatClick(row, column, seatedPartygoer)}
+              />
             );
           })}
```

Comment:
this was refactored into `src/components/templates/Camp/components/MapRoomsOverlay.tsx`